components/places/ios/Places/RustPlacesAPI.h

```diff
@@ -60,15 +60,6 @@ void places_note_observation(PlacesConnectionHandle handle,
                              const char *_Nonnull observation_json,
                              PlacesRustError *_Nonnull out_err);
 
-char *_Nullable places_query_autocomplete(PlacesConnectionHandle handle,
```

Comment:
those weren't used anywhere as far as I can tell, I searched in our repo and in firefox ios

Reply:
yeah, ios only uses our bookmarks but not history, so this makes sense to me.